homeassistant/components/epson/__init__.py

```diff
@@ -3,11 +3,11 @@
 import logging
 
 from epson_projector import Projector
-from epson_projector.const import POWER
+from epson_projector.const import POWER, STATE_UNAVAILABLE
```

Comment:
I would recommend to rename this variable to better differentiate between the HA constant and this library one.
```suggestion
from epson_projector.const import POWER, STATE_UNAVAILABLE as EPSON_STATE_UNAVAILABLE
```